syntax = "proto3";

service EchoService {
    rpc SendEcho (Echo) returns (EchoResponse) {}
    rpc SendEchoReverse (Echo) returns (EchoResponse) {}
}

message Echo {
    string Message = 1;
}

message EchoResponse {
    string Message = 1;
}
